workspaces/autopts-zephyr-22/autopts-zephyr-22.pqw6

```diff
@@ -1,4 +1,4 @@
-<WORKSPACE_INFORMATION NAME="autopts-zephyr-22" PATH="C:\msys64\home\michaln\auto-pts\workspaces\autopts-zephyr-22" CREATED_WITH="Profile Tuning Suite, 7.6.0, 15" BD_ADDR="000000000000" QDID="145446">
+<WORKSPACE_INFORMATION NAME="autopts-zephyr-22" PATH="C:\Users\codecoup\auto-pts\workspaces\autopts-zephyr-22\autopts-zephyr-22" CREATED_WITH="Profile Tuning Suite, 7.6.0, 15" BD_ADDR="000000000000" QDID="146022">

```

Comment:
Why is there a hardcoded path here?

Reply:
Forgot to remove it. Should be fine now.